schemas/styles/csl.rnc

```diff
@@ -647,10 +647,11 @@ div {
       ## Specify verbatim text.
       attribute value { text }
     | ((attribute variable { variables.numbers | variables.strings },
-        [ a:defaultValue = "long" ] attribute form { "short" | "long" })
+        [ a:defaultValue = "long" ]
+        attribute form { "short" | "long" }?)
        | (attribute variable { variables.titles },
           [ a:defaultValue = "long" ]
-          attribute form { "short" | "long" | "sub" | "main" })?)
+          attribute form { "short" | "long" | "sub" | "main" }?)?)
```

Comment:
I'm a bit confused by this one. I thought "sub" and "main" on `@form` will be added in 1.1, not already in 1.0.2. Maybe we got confused with the different branches?

Reply:
Yes, these were removed from 1.0.2

Reply:
I don't recall exactly what the status of master vs 1.0 branches were once we finished. @bdarcus how does 1.0 run in your tests?

Reply:
Oh, I was using master here. I guess I should have clarified that.

Was that wrong?

Reply:
I think 1.0 (and the tag) was the official "1.0.2"

Reply:
I'm recalling our frustration managing the various branches

Reply:
> Oh, I was using master here. I guess I should have clarified that.
> 
> Was that wrong?

Well, sort of. There's nothing wrong with fixing bugs on that branch, but at least it comes as no suprise that validating 1.0.1/1.0.2 styles fails with that branch:

![grafik](https://user-images.githubusercontent.com/4668198/121140159-d2c8d400-c839-11eb-921e-1a3e2a916f48.png)

Reply:
Hmm, I've got version 1.1 even with v1.0.2 checked out. It looks like that happened somewhere between v1.0.2-pre.2 and v1.0.2.  (Current styles seem to be valid with v1.0.2-pre.2, but Atom is throwing millions of errors with v1.0.2)

Reply:
Hmm ...can I ask you two to figure out the best way forward here? It sounds like maybe we should close this PR and branch and start over?

I had a sudden death in the family this weekend, so I'm a bit distracted.

Reply:
I'll try to figure out what was happening between v1.0.2-pre.2 and v1.0.2.

Reply:
I'll open a new PR. Let's close this one for now.